### ISSUE TITLE
Any way to contact you?

### ISSUE THREAD

**Issue by Chinafreak** [Aloshi/EmulationStation#493]
Hello @Aloshi ,
I know this is not the correct place to ask you, but I've no any chance to contact you. You didn't answer me the email for 1 week. :/
I've questions and I'd glad if you could watch into the emails and answer it. :)

- Yan

**Comment by robertybob** [comment]
Aloshi doesn't have time to maintain Emulationstation , so you aren't going to get a reply from him.

**Comment by robertely** [comment]
@robertybob Yea, so about that. Is this project just not maintained any more? Because it smells like it's getting there.

**Comment by Chinafreak** [comment]
and writing on forum is inpossible, reason of spams. :/

**Comment by gonzo-the-great** [comment]
yup sadly, I think his new job is more demanding than before, so what free time he has is shared with his family, so sadly it had a knock on effect here.
but as we all know this is just a hobby, not an income, sadly

**Comment by robertely** [comment]
FYI: the Retro pi people have a fork they are maintaining. It's a little half-cocked, but it's working.

**Comment by joolswills** [comment]
it only has some changes for configuring more buttons etc for our auto configure functionality. Nothing else (yet), the recall box fork has a lot more stuff - which we may pull into ours too)

**Comment by Aloshi** [comment]
Hey, I'm still alive...I pretty much stopped checking my e-mail because I started getting more messages than I could deal with (sitting at 2,328 unread) and once they piled up, I felt guilty for not responding to the help posts/feature requests/bug reports/etc. It's usually something I've already answered somewhere else/a great idea that I don't have time to implement/an obscure bug I don't know how to fix, so when I got busy I stopped paying attention.

I also go to work and debug a large C++ codebase for 7 hours, and when I come home I don't feel like debugging another large C++ codebase for hours. So I haven't been working on ES.

...so, uh, what's up?

**Comment by robertybob** [comment]
Glad to hear you're alive ! :)

**Comment by Chinafreak** [comment]
yay \o/ all what I want to hear is that you're still alive. :) 
And holy shit, ~2328 unread emails? Poor Aloshi. But my email is not a "help posts/feature requests/bug reports", my email is a totally different. ^^. 

Anyways, if you can't watch all the emails, then: Any way to contact you private?

**Comment by Aloshi** [comment]
Can you resend the e-mail with "Chinafreak" in the title? I'll follow up with you there.

**Comment by Chinafreak** [comment]
Done. Thank you :)

**Comment by jammet** [comment]
I know this is not the place, but since everything about this entire post is completely offtopic anyway:

What's the situation with Recalbox? Is 3.2.11 the latest version of Recalbox and will that Online Update feature actually work? Will it include updating the built-in Kodi?

And this Retropie thing, last time I used it, it was lagging behind recalbox for features, polish and stability, has that changed since January?

Sorry with all the questions...

**Comment by digitalLumberjack** [comment]
Hi everyone. @Aloshi nice to hear from you. @joolswills good news you have a configuration tool for retropie :)
@jammet it may not be the best place to discuss about that, but the update will not be available between a 3.2.X and 3.3.0 next release.
@Chinafreak i see in your repositories your are working on a mobile edition of emulationstation, really good idea !

Have a good day everyone !

**Comment by Chinafreak** [comment]
Oh shit, somebody just found it. Now it's time for plan B!
